### PR TITLE
6 get article comments

### PR DESCRIPTION
**Pull request by kenchan0824**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -107,3 +107,43 @@ describe("GET /api/articles", () => {
...
+            article_id: expect.any(Number),
+            author: expect.any(String),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+          });
+        });
+        expect(body.comments).toBeSorted({ descending: true, key: "created_at" });
+      });
+  });
+
```
Comment (NRMSMITH) [inline __tests__/app.test.js:140]: Another 200 test for if the article exists but there are no comments (should respond with an empty array)

**Review by NRMSMITH — CHANGES_REQUESTED** [review]
Just one test to add :)

File: __tests__/app.test.js [hunk 2]
```diff
@@ -107,3 +107,52 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  it("200: response with empty array if article exists but has no comment", () => {
```
Comment (itskatherine) [inline __tests__/app.test.js:112]: Brill 👍

**Review by itskatherine — APPROVED** [review]
Great stuff Kendrick good to merge 👍
